src/Calculator.jsx

```diff
@@ -67,9 +69,9 @@ class Calculator extends Component {
         return <table className="calculator-table">
             <thead>
                 <tr>
-                    <th>Ingredient</th>
-                    <th>Per 100g</th>
-                    <th>Grams</th>
+                    <th><IoFastFood /> Ingredient</th>
+                    <th><FaWeightHanging /> Per 100g</th>
+                    <th><FaWeight /> Grams</th>
                     <th>Total</th>
```

Comment:
I think that it would be more consistent if we add an icon to the total column as well. Feel free to choose another column that you like more.
```suggestion
                   <th><FaCalculator /> Total</th>
```

Reply:
Amazing idea! I will add it to the code. :)